### PR TITLE
Create a mechanism to connect to the database via connection string a…

### PR DESCRIPTION
**Pull request by Setup1337**
…nd create first migration with the user table and project table

### CODE REVIEW

File: backend/Tasque.Core.Common/Entities/Project.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+public class Project
+{
+    public int Id { get; set; }
+    [Required]
+    public string Name { get; set; } = "";
+    public string Description { get; set; } = "";
+    public int AuthorId { get; set; }
+    public DateTime CreatedAt { get; set; }
+    public DateTime UpdatedAt { get; set; }
+}
```
Comment (FeloniukM) [inline backend/Tasque.Core.Common/Entities/Project.cs]: Github loves a space at the end of the file

File: backend/Tasque.DAL/DataContext.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        
+    }
+    protected override void OnModelCreating(ModelBuilder modelBuilder)
+    {
+        modelBuilder.UseSerialColumns();
+    }
+
+    public DbSet<User> Users { get; set; }
+    public DbSet<Project> Projects { get; set; }
+}
```
Comment (FeloniukM) [inline backend/Tasque.DAL/DataContext.cs]: same

File: backend/Tasque.Core.Common/Entities/Project.cs (hunk 1 above)
Comment (Setup1337) [reply backend/Tasque.Core.Common/Entities/Project.cs]: Done

File: backend/Tasque.DAL/DataContext.cs (hunk 2 above)
Comment (Setup1337) [reply backend/Tasque.DAL/DataContext.cs]: Done

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by FeloniukM — APPROVED** [review]

**Comment by ihor-il** [comment]
It could be nice to fix build warnings on nullable DbSets

**Pull request merged**